lib/experimental/Widgets/Content/TasksList/TaskItem/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Badge } from "@/components/Information/Badge"
+import { Counter } from "@/experimental/Information/Counter"
+import { DottedCircle, InProgressTask } from "@/icons"
+import { useCallback } from "react"
+
+export type TaskStatus = "in-progress" | "due" | "no-due"
+export interface Task {
+  id: number | string
+  text: string
+  badge?: { text: string; isPastDue?: boolean }
+  counter?: number
+}
+
+export type TaskItemProps = {
+  task: Task
+  status: TaskStatus
+  onClick?: (task: Task) => void
+  hideIcon?: boolean
+}
+
+export function TaskItem({
+  task,
+  status,
+  onClick,
+  hideIcon = false,
+}: TaskItemProps) {
+  const handleClick = useCallback(() => {
+    onClick?.(task)
+  }, [onClick, task])
+
+  return (
+    <div
+      className={`flex flex-row items-center gap-2 border border-solid border-transparent p-2 ${
+        onClick
+          ? "cursor-pointer hover:bg-f1-background-tertiary focus:border-f1-background-selected-bold focus:outline-none"
+          : ""
+      }`}
+      style={{ borderRadius: 8 }}
+      onClick={handleClick}
+      tabIndex={onClick ? 0 : -1} // Asegura que sea focusable si tiene onClick
```

Comment:
if it can be clicked, we should use a semantic tag (`a` fits well for our case) or define `role="link"`, preferably the former. This will make tab behave as expected

Reply:
I have a problem with link and a's... they're not navigating throw react-router so I prefer to delegate the onClick behaviour to the parent component

Reply:
maybe we can use event.preventDefault() if we wanna add <a> as a markup

Reply:
As @smalonso we can use <a> but prevent default event and propagate onClick

Reply:
For me is OK! use `<a>` but propagate onClick to the parent. @nlopin ?

Reply:
yep, this should work without any issues